app/src/main/res/layout/fragment_compose.xml

```diff
@@ -7,65 +7,60 @@
     android:orientation="vertical"
     tools:context=".fragments.ComposeFragment">
 
+    <Button
+        android:id="@+id/bCamera"
+        style="@style/Widget.Material3.Button.OutlinedButton.Icon"
+        android:layout_width="180dp"
+        android:layout_height="55dp"
+        android:layout_marginStart="8dp"
+        android:fontFamily="@font/copse"
+        android:foregroundGravity="center_vertical"
+        android:text="@string/attach_photo"
+        android:textAllCaps="false"
+        android:textColor="@color/silver"
+        android:textSize="20sp"
```

Comment:
You should look into defining custom styles for attributes that are the same across views.
I think I've seen this same combination of textAllCaps, textColor, and textSize on other buttons too, for example.